### ISSUE TITLE
Disconnected Large amount of Packets

### ISSUE THREAD

**Issue by MousaEqilan** [Suprcode/mir2#853]
when i try open Ranking Dialog and try scroll down get  ( lost connection to server ) and in server exe show  this error 

[CODE=csharp]Disconnected, Large amount of Packets. LastPackets: GetRanking.[/CODE]

also when u try use key x for pick up i get this error

[CODE=csharp]Disconnected, Large amount of Packets. LastPackets: IntelligentCreaturePickup,Run.[/CODE]

i think the bug on original code for the Packet detection

**Comment by Suprcode** [comment]
Pretty sure both of these issues were fixed long ago

**Comment by MousaEqilan** [comment]
don't think so as im follow every update on here and this still give me this

**Comment by Suprcode** [comment]
please download the original source code and try it. We can't base bug reports on your own files

**Comment by MousaEqilan** [comment]
lol bro im use original files  with follow all update and don't tell me there no bugs if you remember old crash's been reported and you said not from the code then all crashed been from original code , all been fixed 

if you remember **player null** issues etc....  this been from original  bugs

**Comment by Suprcode** [comment]
you're raising a bug on your own files by the assumption you've copied over all changes correctly. Makes it very difficult to verify if its an existing bug with the untouched files or not.

Ranking packet spam was adjusted in this commit. so i'm surprised its still an issue.

https://github.com/Suprcode/mir2/commit/7c9cb84af8a8ea24423f5460b0fc0cf817f3db0f

so please verify these are actual issues in the main source code.

**Comment by MousaEqilan** [comment]
yes they still in main files not my files as my files its main code also 

try it online put it live files and let ppl connect to it and see the bug yourself mate to believe this shit
